src/js/big-play-button.js

```diff
@@ -8,16 +11,20 @@
  * @class
  * @constructor
  */
-vjs.BigPlayButton = vjs.Button.extend();
+var BigPlayButton = Button.extend();
```

Comment:
We should consider switching from jshint to eslint. It's support for ES6 and above is far superior currently.